po/installation/ar.po

```diff
@@ -160,7 +160,7 @@ msgid ""
 "differ from those in the manual.\n"
 msgstr ""
 "لا يستوفي الكمبيوتر الخاص بك كل المتطلبات \n"
-"للتثبيت الرسومي. هناك إما أقل من 1% في ذاكرة MB\n"
+"للتثبيت الرسومي. هناك إما أقل من 1% في ذاكرة %1 MB\n"
```

Comment:
Good catch, but your correction is still wrong because it includes both %1 and 1%. I'll fix it.